### ISSUE TITLE
modifying [cmder_prompt_config.lua] this file cannot be changed "λ"

### ISSUE THREAD

**Issue by SkinAir** [cmderdev/cmder#2716]
![图片](https://user-images.githubusercontent.com/71880910/177821383-eeb4802f-3dc2-4855-9b06-9fa9f51cf872.png)
![图片](https://user-images.githubusercontent.com/71880910/177821814-4838690f-afb4-4d22-9684-097b02d79421.png)

**Comment by SkinAir** [comment]
Found many articles, but failed to modify them“ λ "And line breaks, thank, help me answer

**Comment by SkinAir** [comment]
> 找了很多文章，但是没能修改“λ”和换行，谢谢，帮我解答

include：https://github.com/cmderdev/cmder/issues/2348

**Comment by daxgames** [comment]
I don't understand the issue.  I change this file and my prompt changes.

Please clarify.

**Comment by SkinAir** [comment]
> I don't understand the issue. I change this file and my prompt changes.
> 
> Please clarify.

thank you.

**Comment by DRSDavidSoft** [comment]
The issue is that `cmder_prompt_config.lua` is for cmd.exe/clinks shell while for bash, `PS1` needs to be edited instead:

https://github.com/cmderdev/cmder/blob/711fe2895e5331e183ee72c200a3f67349f1fa12/vendor/git-prompt.sh#L71

Currently it seems that the `~/.config/git/git-prompt.sh` file needs to be edited/created and lines 41-71 be copied into it in order to change the prompt.

@daxgames The documentation needs to be updated in order to make this clear that the `.lua` files affect only the clink shell and not the bash shell.  In the meantime, I added it to the [wiki](https://github.com/cmderdev/cmder/wiki/Customization#bash-prompt-symbol).

**Comment by daxgames** [comment]
@DRSDavidSoft I did not even know that was a thing.  It is more a `Git-for Windows` convention since [cmder/vendor/git-prompt.sh](https://github.com/cmderdev/cmder/blob/711fe2895e5331e183ee72c200a3f67349f1fa12/vendor/git-prompt.sh) most likely came from there and just we changed the  `$` to `λ` character.

I think it would be better to take advantage of this, now that we know about it, and make a more Cmder like solution and put user prompt configurability like we have for cmd/lua prompts into `$CMDER_ROOT/config/cmder_prompt_config.sh`

**Comment by DRSDavidSoft** [comment]
@daxgames Good idea!  In fact I think if we could leverage the same `.lua` files for both Bash and Powershell, it'd be awesome to share the `cmder_prompt_config` file across different shells.  This will however require a lua parser for both of those environments.  In #1695, I proposed a shared config storage (e.g. ala .`ini`-style files, or even `json`) that could be parsed by all three environments, or so.

Other than that, we could use separate `cmder_prompt_config` . `sh`, `lua`, `ps1` files, too.

**Comment by chrisant996** [comment]
How about a simple `X = Y` file format that can be read (or maybe included) but all the various scripting languages?

That way the configuration could be independent from script languages.

**Comment by daxgames** [comment]
@chrisant996 exactly what I was thinking.

**Comment by chrisant996** [comment]
Clink v1.4.8 looks like it may have solved the width problems with the lambda character in the prompt.
